src/runcode-func.jl

```diff
@@ -340,17 +331,20 @@ function affect!(
         sw.eruption = 1
         rho_g0 = eos_g_rho_g(P_0, int.u[2])
         record_erupt_start(int.t, int.u[3], eps_x0, int.u[5], int.u[6], rho_g0, erupt_saved)
-        @info("time: $(int.t), Reached critical pressure and need to start an eruption")
+        @info(
+            "*event idx: $idx\n  time: $(int.t), Reached critical pressure and need to start an eruption"
+        )
     elseif idx == 4
         sw.eruption = 0
         record_erupt_end(int.t, erupt_saved, param)
-        @info("time: $(int.t), Finished an eruption...")
+        @info("*event idx: $idx\n  time: $(int.t), Finished an eruption...")
     elseif idx == 6 || idx == 8
         phase_here = param_saved_var.phase
```

Comment:
@cpaniaguam This part handle the phase conversion when the ode calculation meet the criteria `idx == 6 || idx == 8` (idx is decided by `stopChamber_MT`)